### PR TITLE
Add hackathon details

### PR DESCRIPTION
**Pull request by dbarrosop**
@mirceaulinic @ktbyers @GGabriele @ubaumann @mierdin

**What do I need from you?**

I need a title for your talk and a link to your profile or webpage. Whatever you prefer.

**Agenda**

If any of you has any time restrictions let me know so I can take it into account. I am also considering if we should start an hour later (5pm UTC). Any thoughts on that?

### CODE REVIEW

File: _posts/2017/2017-11-13-hackathon-2017-defails.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+Be nice, respectful and have fun!
+
+## Agenda
+
+**All times are in UTC**
+
+* Friday
+  * 4.00 pm ~ 7.00pm Kick Off
+    * [Kirk Byers](/ktbyers) - NAPALM update
+    * [Mircea Ulinic](/mirceaulinic) - TBD
```
Comment (mirceaulinic) [inline _posts/2017/2017-11-13-hackathon-2017-defails.md]: napalm-logs

File: _posts/2017/2017-11-13-hackathon-2017-defails.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+title: "Hackathon 2017 Details"
+author: dbarrosop
+description: "Details about the hackathon 2017"
+tags: ["news"]
+---
+
+
+## Intro
+
+It's been a year since our last hackathon so it's time for a second take. This year, in addition to giving you a NAPALM update and present a couple of new projects, we will also have a few talks from various other projects leveraging on napalm and a talk from Gabriele Gergino who was the unofficial winner for the non-existent contest we had in the hackathon we held last year :)
```
Comment (mirceaulinic) [inline _posts/2017/2017-11-13-hackathon-2017-defails.md]: Gerbino :P

Comment (GGabriele) [inline _posts/2017/2017-11-13-hackathon-2017-defails.md]: that's Gerbino :p

Comment (dbarrosop) [reply _posts/2017/2017-11-13-hackathon-2017-defails.md]: Damn it! I was so proud I had gotten all the names right in the agenda without having to check...

**Pull request merged**